script/reviewer.go

```diff
@@ -19,11 +30,88 @@ func (r *Reviewer) Review() {
 
 	r.application.Parse(r.gitHub.Issue)
 
-	if isTestingIssue() {
-		if r.application.IsValid() {
-			debugMessage("Application has no problems")
+	status := r.getStatus()
+	r.updateLabels(status)
+	r.createComment(status)
+}
+
+func (r *Reviewer) getStatus() Status {
+	if *r.gitHub.Issue.State == "closed" {
+		return Closed
+	} else if r.gitHub.IssueHasLabel(LabelStatusApproved) {
+		return Approved
+	} else if r.gitHub.IssueHasLabel(LabelStatusReviewing) {
+		return Reviewing
+	} else if r.gitHub.IssueHasLabel(LabelStatusInvalid) {
+		return Invalid
+	} else {
+		return New
+	}
+}
+
+func (r *Reviewer) createComment(status Status) {
+	title := ""
+	body := ""
+	details := fmt.Sprintf("<details>\n<summary>Application data...</summary>\n\n```json\n%s\n```\n</details>", r.application.GetData())
+	// TODO: replace FILE_NAME with Application.FileName once available
+	dataPath := fmt.Sprintf("https://github.com/1Password/1password-teams-open-source/blob/main/data/%s", "FILE_NAME")
```

Comment:
Is this `TODO` for this PR or for the future?

Reply:
Future, once https://github.com/1Password/1password-teams-open-source/pull/925 is merged

Reply:
Update: that PR was merged so I've updated!